pages/search.js

```diff
@@ -20,7 +21,7 @@ export async function getStaticProps() {
     tags: [],
   };
   try {
-    data.users = await getUsers({ cards: true });
+    data.users = await getUsers({ cards: false });
```

Comment:
there is no need to include the object as a parameter as cads false is the default.